dredd/api-description.yml

```diff
@@ -989,7 +990,7 @@ definitions:
         type: array
         items:
           type: string
-          description: Country codes
+          description: Country names
```

Comment:
Does this still pass `en`, etc?

Reply:
The documentation was just incorrect.
So, from the updated `Series.to_json`:
```py
        data['countries'] = self.countries  # e.g. ['ITALY', 'FRANCE']
        data['countryCodes'] = self.country_codes  # e.g. ['it', 'fr']
```